azure-pipelines.yml

```diff
@@ -208,26 +208,6 @@ jobs:
       includePatterns:
         - "/sw/device/***"
 
-- job: chip_earlgrey_verilator
```

Comment:
It's a bit of a pity not to keep this as a separate job: it means that we won't be able to have two downstream "now run the tests" jobs without having to build the verilator simulation twice. Is there any way to hand off the built artefacts between jobs in the way that we were doing before?

Reply:
I think we'll get that with Bazel caching, once it's implemented (there's a TODO: below about this). We'll just need a 'build verilator' job that the two current test jobs depend upon so they don't both start building in parallel because there's nothing in the cache yet.

Before that happens we'll end up adding a bit of extra load to the ci-public agent pool but the current build looks to be taking around 10 minutes so hopefully won't have a dramatic effect.

Reply:
Yes, the plan here ultimately is to first build the tests and the verilated model using a bazel disk cache, upload this cache as an artifact (similar to uploading build-bin), and download it in subsequent steps that run the tests. We should be able to achieve a similar setup to the previous CI arrangement. Note that this may be somewhat contingent on figuring out #9653.